Tests/Common/SymbolTests.cs

```diff
@@ -556,6 +557,82 @@ public void TestIfFridayLastTradingDayIsHolidaysThenMoveToPreviousThursday()
             Assert.AreEqual(thursdayBeforeGoodFriday, OptionSymbol.GetLastDayOfTrading(symbol));
         }
 
+        [Test]
+        public void NegativeStrikePriceRoundTrip()
+        {
+            var future = Symbol.CreateFuture(
+                "CL",
+                Market.NYMEX,
+                new DateTime(2020, 5, 20));
+
+            var option = Symbol.CreateOption(
+                future,
+                Market.NYMEX,
+                OptionStyle.American,
+                OptionRight.Call,
+                -50,
+                new DateTime(2020, 4, 16));
+
+            Assert.AreEqual(-50, option.ID.StrikePrice);
+
+            var properties = (ulong)typeof(SecurityIdentifier)
+                .GetField("_properties", BindingFlags.NonPublic | BindingFlags.Instance)
+                .GetValue(option.ID);
```

Comment:
Suggest use `ToString()` and later the `Parse()`. Let's add asserts on all the other properties too, so we are sure it's not breaking 👍